source/includes/api/_areas_v2.md

```diff
@@ -0,0 +1,280 @@
+deforestationAlerts  | If the area is intended to subscribe to deforestation alerts. Defaults to false.                   | Boolean | true       |
+monthlySummary       | If the area is intended to subscribe to monthly summaries. Defaults to false.                      | Boolean | true       |
+email                | Email to be provided to the subscription.                                                          | String  | youremail@resourcewatch.org |
+webhookUrl           | Webhook URL to be provided to the subscription (only used in case the email is not set).           | String  | https://www.google.com/ |
+status               | The status of the area - either 'saved' or 'pending'. Read-only attribute.                         | String  | 'saved'    |
+subscriptionId       | The ID of the subscription associated with this area. Read-only attribute.                         | String  | 5e4d7c47dd8fa31290d548ae |
+
+**Implementation details**
+
+POST of a new area always starts by creating the area and, taking into account the area attributes, it might also create a subscription which will then be associated with the area. The area's `subscriptionId` attribute will then be updated with the id of the created subscription if that's the case. The created area is then returned.
+
+The subscription is created only if the area has selected set to `true` at least one of the following attributes:
+
+* deforestationAlerts
+* fireAlerts
+* monthlySummary
+
+## Updating an area
+
+```shell
+curl -X PATCH https://api.resourcewatch.org/v2/area/:id
+-H "Authorization: Bearer <your-token>" \
+-H "Content-Type: application/json" -d \
+ '{
+    "name": "Example area",
+    "application": "gfw",
+    "image": "",
+    "tags": [],
+    "public": false,
+    "fireAlerts": true,
+    "deforestationAlerts": true,
+    "webhookUrl": "http://example.com",
+    "monthlySummary": true,
+    "email": "youremail@resourcewatch.org",
+    "language": "en"
+}'
+```
+
+> Example response:
+
+```json
+{
+    "data": {
+        "type": "area",
+        "id": "5e4d7c47ef240412c2d56a78",
+        "attributes": {
+            "name": "Example area",
+            "application": "gfw",
+            "wdpaid": null,
+            "userId": "5e2f0eaf9de40a6c87dd9b7d",
+            "createdAt": "2020-02-19T18:19:51.485Z",
+            "image": "",
+            "datasets": [],
+            "tags": [],
+            "status": "pending",
+            "public": false,
+            "fireAlerts": true,
+            "deforestationAlerts": true,
+            "webhookUrl": "http://example.com",
+            "monthlySummary": true,
+            "subscriptionId": "5e4d7c47dd8fa31290d548ae",
+            "email": "youremail@resourcewatch.org",
+            "language": "en"
+        }
+    }
+}
+```
+
+Use this endpoint to update an existing area. This endpoint requires authentication.
+
+This endpoint supports the same request body parameters as the `POST area` endpoint.
```

Comment:
copy paste the table here - you never know when they'll start evolving in parallel